content/scala-cli.md

```diff
@@ -165,3 +171,539 @@ Scala CLI has many alternatives. The most obvious ones are sbt, Mill, or other b
 ## FAQ
 
 This section will probably initially be empty. As discussions on the proposal progress, it is likely that some questions will come repeatedly. They should be listed here, with appropriate answers.
+
+# Scala Runner Specification
+
+This section describes proposed Scala Runner specification and was generated from Scala CLI documentation. It contains `MUST` have and `SHOULD` have commands (each with complete list of MUST have and SHOULD have options) followed by a list of using directives.
+
+## Scalac options
+
+Scala Runner MUST support following options from Scala Compiler directly:
+ - `-encoding`
+ - `-release`
+ - `-color`
+ - `-nowarn`
+ - `-feature`
+ - `-deprecation`
+
+
+ Additionally, all options that start with:
+- `-g`
+- `-language`
```

Comment:
I think `-source` should also be adopted by default, currently you must do e.g. `-O -source:3.2`